src/phoenix/server/api/helpers.py

```diff
@@ -0,0 +1,40 @@
+from collections import defaultdict
+from typing import Any, Dict, Iterable, List, Mapping
+
+from phoenix.core.model_schema import PRIMARY, REFERENCE, DatasetRole, EventId, Model
+from phoenix.metrics import Metric
+from phoenix.server.api.types.DatasetValues import DatasetValues
+
+
+def ensure_list(obj: Any) -> List[Any]:
```

Comment:
```suggestion
def ensure_list(obj: Union[List[T], Iterable[T]], None, Unset) -> List[T]:
```
Does this work so you can enforce the input a tad better